### DISCUSSION TITLE
Plugin development

### DISCUSSION THREAD

**Discussion by kpalang** [nextgenhealthcare/connect#4483] [Show and tell]
Hello everyone!

So thought I'd try this feature out by having a thread of something I've started to quite enjoy with Connect, Plugin development.

First I'd like to share a Maven Repository hosting the connectors and plugins from this same GitHub repository.
This is intended as an unofficial official repository for jar files and will receive almost real-time updates whenever there's a new release of Connect.
https://maven.kaurpalang.com 

Here is a Maven plugin to aid with the generation of the plugin.xml file.
https://github.com/kpalang/mirth-plugin-maven-plugin

And here is a sample structure to showcase the previous Maven plugin
https://github.com/kpalang/mirth-sample-plugin

**Comment by dfauraz** [comment]
Interesting but **https://nexus.kaurpalang.com** cannot be reached, unfortunately

**Reply by jonbartels** [reply]
`https://maven.kaurpalang.com/` is the correct URL. You can see it in the POM at https://github.com/kpalang/mirth-sample-plugin/blob/main/pom.xml

**Reply by kpalang** [reply]
Good point! I've updated the url.
